### ISSUE TITLE
VM looks blocked and not accessible. 

### ISSUE THREAD

**Issue by bastos1971** [canonical/multipass#3138]
Hello

I have an issue with my multipass.  I cannot launch it anymore from my Mac.

I have uninstalled/reinstalled multipass without any success (brew and classic pkg)
 
I could erase all by VM with the command line except 2 : docker and primary which seam not respond and block all multipass : 

% multipass version
multipass   1.12.0+mac
multipassd  1.12.0+mac
% multipass list
Name                    State             IPv4             Image
primary                 Running           192.168.64.14    Ubuntu 22.04 LTS
docker                  Running           192.168.64.12    Ubuntu 22.04 LTS
docker1                 Deleted           --               Not Available

% multipass shell docker
shell failed: qemu-system-aarch64: Failed to load snapshot: File too large

% multipass shell docker
shell failed: ssh connection failed: 'Timeout connecting to 192.168.64.12'

% multipass stop docker
stop failed: cannot connect to the multipass socket

 % ping 192.168.64.12
PING 192.168.64.12 (192.168.64.12): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2
Request timeout for icmp_seq 3
ping: sendto: No route to host
Request timeout for icmp_seq 4

~ % multipass stop docker
stop failed: cannot connect to the multipass socket

It seams that these 2 VM block all my Multipass, even graphical icon that it not shown any more.

Any idea ? I just want to reinstall multipass from scratch and use multipass. It looks like even after a proper uninstall (Brew or Cleaner and Uninstall app) some file remind in my mac an prevent me to reinstall all from scratch. 

I am using MacBook Pro M1 Pro 2021. Ventura 13.4.1

Thanks

Bastos

**Comment by townsend2010** [comment]
Hi @bastos1971!

Sorry for your troubles.  The `Failed to load snapshot: File too large` error looks very interesting.  I have never seen that error before, so I would like to ask if you could provide some more info before we dive into trying to fix your install.

First, could you please provide [logs](https://multipass.run/docs/accessing-logs#heading--macos)?

Second, could you please run the following commands and provide the output?  I need these in order to help further diagnose the issue.
1. `% sudo /Library/Application\ Support/com.canonical.multipass/bin/qemu-img info /var/root/Library/Application\ Support/multipassd/qemu/vault/instances/docker/ubuntu-22.04-server-cloudimg-arm64.img`
2. `% sudo /Library/Application\ Support/com.canonical.multipass/bin/qemu-img info /var/root/Library/Application\ Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img`
3. `% ps -aef | grep qemu`

Thanks!

**Comment by bastos1971** [comment]
Hello Chris, 

Thanks for the quick answer. 

Please find de log attached and the output of your 3 questions

Many thanks and best regards

Bastien 

Answer 1)

image: /var/root/Library/Application Support/multipassd/qemu/vault/instances/docker/ubuntu-22.04-server-cloudimg-arm64.img
file format: qcow2
virtual size: 40 GiB (42949672960 bytes)
disk size: 9.57 GiB
cluster_size: 65536
Snapshot list:
ID        TAG               VM SIZE                DATE     VM CLOCK     ICOUNT
1         suspend          1.25 GiB 2023-06-22 14:25:15 77:18:01.170           
Format specific information:
    compat: 0.10
    compression type: zlib
    refcount bits: 16
Child node '/file':
    filename: /var/root/Library/Application Support/multipassd/qemu/vault/instances/docker/ubuntu-22.04-server-cloudimg-arm64.img
    protocol type: file
    file length: 13.8 GiB (14799536128 bytes)
    disk size: 9.57 GiB

Answer 2

fg: no current job
bastien@bastiens-MBP Multipass % sudo /Library/Application\ Support/com.canonical.multipass/bin/qemu-img info /var/root/Library/Application\ Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img
image: /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img
file format: qcow2
virtual size: 5 GiB (5368709120 bytes)
disk size: 5.06 GiB
cluster_size: 65536
Format specific information:
    compat: 0.10
    compression type: zlib
    refcount bits: 16
Child node '/file':
    filename: /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img
    protocol type: file
    file length: 5.19 GiB (5575016448 bytes)
    disk size: 5.06 GiB

3) 

ps -aef | grep qemu
    0   902     1   0 Dim06   ??         4:49.84 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:87:84:11 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 2 -m 4096M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/cloud-init-config.iso
    0  5662     1   0 Dim06   ??         4:22.07 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:87:84:11 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 2 -m 4096M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/cloud-init-config.iso -loadvm suspend -machine virt-8.0
    0  7403     1   0 Dim06   ??         4:23.44 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:87:84:11 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 2 -m 4096M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/cloud-init-config.iso -loadvm suspend -machine virt-8.0
    0  8776     1   0 Dim06   ??         4:19.07 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:87:84:11 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 2 -m 4096M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/cloud-init-config.iso -loadvm suspend -machine virt-8.0
    0 11351     1   0 Dim06   ??         4:05.89 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:87:84:11 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 2 -m 4096M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/cloud-init-config.iso -loadvm suspend -machine virt-8.0
    0 11356     1   0 Dim06   ??         6:03.48 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:18:12:87 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/cloud-init-config.iso -loadvm suspend -machine virt-8.0
    0 11669     1   0 Dim06   ??         2:24.22 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:80:84:cc -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/faithful-gobbler/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/faithful-gobbler/cloud-init-config.iso
    0 12170     1   0 Dim06   ??         6:03.77 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:18:12:87 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/cloud-init-config.iso
    0 13994     1   0 Dim06   ??         5:43.67 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:18:12:87 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/cloud-init-config.iso -loadvm suspend -machine virt-8.0
    0 14011     1   0 Dim06   ??         1:26.81 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:87:84:11 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 2 -m 4096M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/docker1/cloud-init-config.iso
    0 18929     1   0 Dim06   ??         1:03.00 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:18:12:87 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/cloud-init-config.iso
    0 20451     1   0 Dim07   ??         1:26.81 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:18:12:87 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/cloud-init-config.iso
    0 20966     1   0 Dim07   ??       550:10.48 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:18:12:87 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/cloud-init-config.iso
    0 21052     1   0 Dim07   ??       549:55.37 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:18:12:87 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/cloud-init-config.iso
    0 23892     1   0 Dim07   ??       541:41.00 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:18:12:87 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/cloud-init-config.iso
    0 24689 24682   0 Dim07   ??       538:29.66 /Library/Application Support/com.canonical.multipass/bin/qemu-system-aarch64 -machine virt,gic-version=3 -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu host -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:18:12:87 -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 1024M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/primary/cloud-init-config.iso
  501 13494 12418   0 11:46   ttys001    0:00.00 grep qemu
bastien@bastiens-MBP ~ % 

[multipassd.log](https://github.com/canonical/multipass/files/11880238/multipassd.log)

**Comment by townsend2010** [comment]
Hi @bastos1971!

Thanks for providing the requested info.  I've looked at everything you posted along with QEMU source code and it's still not clear how this is occurring.  I would like now to try to help you recover from this situation.

Please try the following:
1. `% sudo killall -9 qemu-system-aarch64`
2. `% sudo launchctl unload /Library/LaunchDaemons/com.canonical.multipassd.plist`
3. `% launchctl unload /Library/Application\ Support/com.canonical.multipass/Resources/com.canonical.multipass.gui.autostart.plist`
4. `% sudo /Library/Application\ Support/com.canonical.multipass/bin/qemu-img snapshot -d suspend /var/root/Library/Application\ Support/multipassd/qemu/vault/instances/docker/ubuntu-22.04-server-cloudimg-arm64.img`
5. `% sudo launchctl load -w /Library/LaunchDaemons/com.canonical.multipassd.plist`

Please let me know how it goes.  Thanks!

**Comment by bastos1971** [comment]
Hello

Thanks for answer. It has worked. 


I have launched your commands. There were OK until : 

sudo /Library/Application\ Support/com.canonical.multipass/bin/qemu-img snapshot -d suspend /var/root/Library/Application\ Support/multipassd/qemu/vault/instances/docker/ubuntu-22.04-server-cloudimg-arm64.img
qemu-img: Could not delete snapshot 'suspend': Failed to free the cluster and L1 table: File too large

Even if the command failed, I have tried the "Multipass list » which were much more faster than yesterday. 

Primary VM was stopped and docker starting

multipass list
Name                    State             IPv4             Image
primary                 Stopped           --               Ubuntu 22.04 LTS
docker                  Starting          --               Ubuntu 22.04 LTS
docker1                 Deleted           --               Not Available

Once docker was started again, It has blocked again all VM. I have stopped, deleted  and prune the docker, it took a while, but I succeeded.

I have launched again docker from scratch and it has succeed. 

multipass launch docker
You'll need to add this to your shell configuration (.bashrc, .zshrc or so) for
aliases to work without prefixing with `multipass`:

PATH="$PATH:/Users/bastos/Library/Application Support/multipass/bin"
Folder "/Users/bastien/multipass/docker" already exists.
Mounted '/Users/bastien/multipass/docker' into 'docker:docker'
Launched: docker

 multipass list
Name                    State             IPv4             Image
primary                 Stopped           --               Ubuntu 22.04 LTS
docker                  Running           192.168.64.18    Ubuntu 22.04 LTS
                                          172.17.0.1
multipass shell docker

Welcome to Ubuntu 22.04.2 LTS (GNU/Linux 5.15.0-75-generic aarch64)

 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage

  System information as of Tue Jun 27 22:58:03 CEST 2023

  System load:              0.1513671875
  Usage of /:               6.3% of 38.59GB
  Memory usage:             9%
  Swap usage:               0%
  Processes:                115
  Users logged in:          0
  IPv4 address for docker0: 172.17.0.1
  IPv4 address for enp0s1:  192.168.64.18
  IPv6 address for enp0s1:  fdfc:5640:8938:bd13:5054:ff:fec2:2ecd


Expanded Security Maintenance for Applications is not enabled.

3 updates can be applied immediately.
3 of these updates are standard security updates.
To see these additional updates run: apt list --upgradable

Enable ESM Apps to receive additional future security updates.
See https://ubuntu.com/esm or run: sudo pro status

That’s it !!

I use 2 MacBook. I have Multipass on both of them, But all the issues I have are occurs on the M1 Pro and not on the Intel. 

Many thanks for the support

Bastien
